octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/comment/job/GetCommentsJob.java

```diff
@@ -29,7 +29,7 @@
 
 public class GetCommentsJob extends Job {
 
-    private static final List<Entity> noCommentsEntites = Arrays.asList(Entity.TASK);
+    private static final List<Entity> noCommentsEntites = Arrays.asList();
```

Comment:
is this used somewhere>. if not, remove it

Reply:
it is used in some methods and I thought it would be good for the future, if some new entities would be added